src/config/permissions.php

```diff
@@ -149,4 +149,15 @@
     */
 
     'migration_sub_folder' => '',
+    
+    /*
+    |--------------------------------------------------------------------------
+    | Run seeder in production env
```

Comment:
**Important**

no need to specify production env for this option. just deal depending on the key value directly on any env. so we may modify this line accordingly.